doc/examples.md

```diff
@@ -0,0 +1,174 @@
+# Examples
+
+Here you can find a bunch of simple examples for using ct, with some explanations about what they do. The examples here are in no way comprehensive, for a full list of all the options present in ct check out [configuration.md][1].
+
+## Users and groups
+
+```yaml
+passwd:
+  users:
+    - name: core
+      password_hash: password1
+      ssh_authorized_keys:
+        - key1
+```
+
+This example modifies the existing `core` user, giving it a known password hash (this will enable login via password), and setting its ssh key.
+
+```yaml
+passwd:
+  users:
+    - name: user1
+      password_hash: password1
+      ssh_authorized_keys:
+        - key1
+        - key2
+    - name: user2
+      ssh_authorized_keys:
+        - key3
+```
+
+This example will create two users, `user1` and `user2`. The first user has a password set and two ssh public keys authorized to log in as the user. The second user doesn't have a password set (so log in via password will be disabled), and one ssh key.
```

Comment:
The second user doesn't have a password set (so log in via password will be disabled), but have one ssh key.